plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1222,17 +1222,24 @@ blockly.block.block_predicate_replaceable=Block at position offset by x: %1 y: %
 blockly.block.block_predicate_solid=Block at position offset by x: %1 y: %2 z: %3 is of solid material
 blockly.block.block_predicate_would_survive=%1 would survive at position offset by x: %2 y: %3 z: %4
 blockly.block.feature_block_pile=Block pile feature with block: %1
+blockly.block.feature_coral_claw=Underwater coral claw feature
+blockly.block.feature_coral_mushroom=Underwater coral mushroom feature
+blockly.block.feature_coral_tree=Underwater coral tree feature
+blockly.block.feature_forest_rock=Forest rock feature with block: %1
 blockly.block.feature_huge_brown_mushroom=Huge brown mushroom feature with cap: %1 stem: %2 radius: %3
 blockly.block.feature_huge_red_mushroom=Huge red mushroom feature with cap: %1 stem: %2 radius: %3
+blockly.block.feature_huge_fungus=Huge fungus feature with required ground: %1 stem: %2 hat: %3 decor: %4 planted: %5
```

Comment:
Is the word required needed here? Other features require blocks too so I think it can be left out to make the block a bit shorter

Reply:
Here "required" is for "The fungus can only generate on this block". I think it could be reworded as "only on ground: %1"?

Reply:
I see. Maybe only on ground, and as last parameter after comma, to make it a bit more clear this parameter is condition and not parameter for block to place?